src/main/kotlin/bitbucket/data/Links.kt

```diff
@@ -0,0 +1,10 @@
+package bitbucket.data
+
+import com.fasterxml.jackson.annotation.JsonProperty
+
+data class Links(@JsonProperty("self")private val selfLink: List<Link>) {
```

Comment:
A space between "self")"and "private" is needed

Reply:
I need to merge this PR, because I am going to touch same places in the code. 
Will fix "the space problem" as well :)